src/mlang/backend_compilers/bir_to_c.ml

```diff
@@ -53,10 +55,10 @@ let generate_variable (var_indexes : int Mir.VariableMap.t) (offset : offset)
   | _ ->
       Format.fprintf fmt "TGV[%d/*%s*/%s]" var_index
         (Pos.unmark var.Mir.Variable.name)
-        ( match offset with
+        (match offset with
         | None -> ""
         | GetValue offset -> " + " ^ string_of_int offset
-        | PassPointer -> assert false )
```

Comment:
What is your `ocamlformat` version ? You can get is using `opam list ocamlformat`.